src/runtime_src/core/common/api/xrt_xclbin.cpp

```diff
@@ -605,6 +619,26 @@ class xclbin_full : public xclbin_impl
       : std::make_pair(nullptr, size_t(0));
   }
 
+  std::vector< std::pair<const char*, size_t> >
```

Comment:
```suggestion
  std::vector<std::pair<const char*, size_t>>
```
C++11 is 10 year old already! :-)